VoodooInput/Trackpoint/TrackpointDevice.cpp

```diff
@@ -19,17 +22,19 @@ UInt32 TrackpointDevice::interfaceID() {
 }
 
 IOItemCount TrackpointDevice::buttonCount() {
-    return 3;
+    return btnCount;
 };
 
 IOFixed TrackpointDevice::resolution() {
-    return (150) << 16;
+    return 150 << 16;
```

Comment:
Is it possible to set the resolution to 400? Some users with ALPS trackpads were complaining that the trackpoint was slower when I switched to VoodooInput's Trackpoint. My Dell has no trackpoint, so I cannot test it.

```suggestion
    return 400 << 16;
```

Reply:
I'd need to test to see how that affects Synaptics, but I probably could. The new code has options for a multiplier too which they could use to speed it up. There's also the mouse settings in system preferences which will mess with trackpoint speed seperately (since it appears as a mouse)